perception/compare_map_segmentation/src/voxel_grid_map_loader.cpp

```diff
@@ -277,10 +298,17 @@ bool VoxelGridStaticMapLoader::is_close_to_map(
 }
 
 VoxelGridDynamicMapLoader::VoxelGridDynamicMapLoader(
-  rclcpp::Node * node, double leaf_size, std::string * tf_map_input_frame, std::mutex * mutex,
+  rclcpp::Node * node, double leaf_size, bool downsize_z_distance_threshold,
+  std::string * tf_map_input_frame, std::mutex * mutex,
   rclcpp::CallbackGroup::SharedPtr main_callback_group)
-: VoxelGridMapLoader(node, leaf_size, tf_map_input_frame, mutex)
+: VoxelGridMapLoader(node, leaf_size, downsize_z_distance_threshold, tf_map_input_frame, mutex)
 {
+  if (downsize_z_distance_threshold) {
+    // reduce voxel leaf size in z axis for low level object detection
+    voxel_leaf_size_z_ = 0.5 * voxel_leaf_size_;
```

Comment:
Is it better to parameterize this 0.5 instead of boolean `downsize_z_distance_threshold` ?

Reply:
@miursh I changed the parameter utility here https://github.com/autowarefoundation/autoware.universe/pull/4243/commits/9dc84436bd9089c7c4aa7d900e54f31c87f89fdd